projects/yarnpkg.com/package.yml

```diff
@@ -0,0 +1,39 @@
+distributable:
+  url: https://github.com/yarnpkg/berry/archive/refs/tags/@yarnpkg/cli/{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: yarnpkg/berry
+
+provides:
+  - bin/yarn
+  # no idea why yarn provides a shortcut of yarnpkg but it errors in mac
+  # - scripts/bin/yarnpkg
+  - bin/yarnpkg
+
+dependencies:
+  nodejs.org: '>=18.3'
+
+build:
+  script: |
+    rm scripts/bin/yarnpkg || true
+    cp -r . {{prefix}}
+    mkdir -p {{prefix}}/bin
+
+    yarnScript=$(cat scripts/bin/yarn)
+    replace="../.."
+    next=..
+    echo "${yarnScript/$replace/$next}" >> {{prefix}}/bin/yarn
+    chmod +x {{prefix}}/bin/yarn
+
+    cp {{prefix}}/bin/yarn {{prefix}}/bin/yarnpkg
+
+    if test "{{hw.platform}}" = "linux"; then
+      $SUDO apt-get --yes install libatomic1
```

Comment:
need help on this im trying to find the first source or how to package this.

Reply:
it's _very_ low level. I believe we require it as a matter of course, so it shouldn't be necessary to install it.